### PR TITLE
Content Safety: Add overload methods.

### PR DESCRIPTION
**Pull request by JieZhou000**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py [hunk 1]
```diff
@@ -18,3 +22,82 @@ def patch_sdk():
     you can't accomplish using the techniques described in
     https://aka.ms/azsdk/python/dpcodegen/python/customize
     """
+
+
+class ContentSafetyClientOperationsMixin(ContentSafetyClientOperationsMixinGenerated):
+    @overload
+    def analyze_text(
```
Comment (msyyc) [inline sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py:29]: Shall it be `analyze_image`?

File: sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py [hunk 2]
```diff
@@ -18,3 +22,82 @@ def patch_sdk():
...
+         will have to context manage the returned stream.
+        :return: AnalyzeTextResult. The AnalyzeTextResult is compatible with MutableMapping
+        :rtype: ~azure.ai.contentsafety.models.AnalyzeTextResult
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        ...
+
+    def analyze_text(self, body: str, **kwargs: Any) -> _models.AnalyzeTextResult:
+        options = _models.AnalyzeTextOptions(text=body)
+        return super().analyze_text(body=options, **kwargs)
```
Comment (msyyc) [inline sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py:52]: If the body only receive one type `str`, I think there is no need to add `@overload` function anymore.

File: sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py [hunk 3]
```diff
@@ -18,3 +22,82 @@ def patch_sdk():
...
+        """
+
+    def analyze_image(self, data: Union[str, bytes], **kwargs: Any) -> _models.AnalyzeImageResult:
+        _image = None
+        if isinstance(data, bytes):
+            _image = _models.ImageData(content=data)
+        else:
+            _image = _models.ImageData(blob_url=data)
+        options = _models.AnalyzeImageOptions(image=_image)
+        return super().analyze_image(body=options, **kwargs)
```
Comment (msyyc) [inline sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py:103]: What if users input `_models.AnalyzeImageOptions(...)` directly?

File: sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py (hunk 2 above)
Comment (JieZhou000) [reply sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py:52]: We have the analyze_text overloads take `_models.AnalyzeTextOptions(...)` as paramter. please check [_operations.py](https://github.com/JieZhou000/azure-sdk-for-python/blob/zhojie/py-0/sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_operations.py) line 305

File: sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py (hunk 3 above)
Comment (JieZhou000) [reply sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py:103]: the same as above.

**Pull request closed without merging**